### PR TITLE
minor clean ups

### PR DESCRIPTION
**Pull request by andersone1**
- Speed up execute data diffs by skipping the diff check if the data frames are identical
- Minor styling changes to v()

### CODE REVIEW

File: R/execute-data-diffs.R [hunk 1]
```diff
@@ -181,22 +181,36 @@ execute_data_diffs <- function(.base_df, .compare_df, .subject_col, .base_from_s
...
+
+      diff.i <-
+        suppressMessages(
+          diffdf::diffdf(
+            base = base_df.i,
+            compare = compare_df.i,
+            suppress_warnings = TRUE,
+            strict_numeric = FALSE,
+            strict_factor = FALSE
+          )
```
Comment (barrettk) [inline R/execute-data-diffs.R:202]: I like this change. Do we want a test for this or no?

File: R/make-v-caption.R [hunk 2]
```diff
@@ -5,12 +5,12 @@
...
-  style <- glue('caption-side: top; text-align: left; color:#0053a0; font-size:{font_size}pt;
+  style <- glue('caption-side: top; text-align: left; color:#FFFFFF; background-color:#5A5A5A; font-size:{font_size}pt;
                 font-weight: bold; position: fixed;left: 12em; top: 1em; padding: 3px')
 
   cap_txt <- if(!is.null(.subject_col)){
     num_subj <- .df %>% dplyr::count(!!!syms(.subject_col)) %>% nrow()
-    glue('Number of Subjects: {num_subj}')
+    glue('N Subjects ({.subject_col}): {num_subj}')
   }else{
     'No subjects detected'
```
Comment (barrettk) [inline R/make-v-caption.R:15]: I just realized I didnt add a `"."` to the `font_size` argument (i.e. `.font_size`). Up to you if you want to fix that, given that it's an internal function

Comment (barrettk) [reply R/make-v-caption.R:15]: Not sure what's going on, but im getting an error when trying to use one of my test datasets from before:

```r
> devtools::load_all(); v(tmp)
ℹ Loading mrgda
Error in writeImpl(text) :
Text to be written must be a length-one character vector
```

**Review by barrettk — CHANGES_REQUESTED** [review]
There seems to be a bug. Will look into it on my end as well

File: R/execute-data-diffs.R (hunk 1 above)
Comment (andersone1) [reply R/execute-data-diffs.R:202]: @barrettk A test would be helpful tbh

**Review by barrettk — APPROVED** [review]
For what it's worth, the bug originated when you have multiple found or specified `.subject_col`. This was previously allowed, but got messed up with the caption change:

```r
> .subject_col
[1] "ID"      "USUBJID"

>   cap_txt <- if(!is.null(.subject_col)){
+     num_subj <- .df %>% dplyr::count(!!!syms(.subject_col)) %>% nrow()
+     glue('N Subjects ({.subject_col}): {num_subj}')
+   }else{
+     'No subjects detected'
+   }

> cap_txt
N Subjects (ID): 49
N Subjects (USUBJID): 49

>   htmltools::tags$caption(
+     style = style,
+     cap_txt
+   )
Error in writeImpl(text) :
Text to be written must be a length-one character vector
```

Will open a separate PR that A) forces `.subject_col` to be a single column if multiple are found/specified, and B) add some more tests for different specifications.

File: R/make-v-caption.R (hunk 2 above)
Comment (andersone1) [reply R/make-v-caption.R:15]: @barrettk Fixed in final commit

**Pull request merged**